compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -135,15 +135,31 @@ public SymbolResolver(CompilerContext context) {
         this.types = Types.getInstance(context);
     }
 
-    public boolean checkForUniqueSymbol(DiagnosticPos pos, SymbolEnv env, BSymbol symbol, int expSymTag) {
+    // TODO : Remove the 'expSymTag'
```

Comment:
is this todo still relevant? `expSymTag` seems to be removed

Reply:
Yes, it is removed.